pyproject.toml

```diff
@@ -12,8 +12,8 @@ classifiers = [
 ]
 description = "A set of functions to reduce the boilerplate required to add observability to your service of module"
 dependencies = [
-    "fastapi[all]<0.99",
-    "typing_extensions<4.6",
+    "fastapi[all]",
+    "typing_extensions",
```

Comment:
Now we're on Python3.10, `ParamSpec` is part of the standard library. You can remove this dependency and do

```python
from typing import ParamSpec
```